pkg/v1/providers/infrastructure-azure/v1.0.0/ytt/base-template.yaml

```diff
@@ -139,7 +144,6 @@ spec:
         - diskSizeGB: 256
           lun: 0
           nameSuffix: etcddisk
-      location: ${AZURE_LOCATION}
```

Comment:
I noticed that this `location` property has been removed from the various AzureMachineTemplate in our ytt templates. I assume this means that it isn't need (should be provided in the MachineDeployment anyway)